src/python/pants/backend/codegen/protobuf/buf.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from typing import List, cast
+
+from pants.core.util_rules.external_tool import TemplatedExternalTool
+from pants.engine.platform import Platform
+
+
+class Buf(TemplatedExternalTool):
+    options_scope = "buf"
+    help = "Professional grade Protocol Buffers (https://github.com/bufbuild/buf)."
```

Comment:
Because we only use this as a linter for now, maybe change this to focus on how Pants uses it?

> A linter for Protocol Buffers (https://github.com/bufbuild/buf).

If you feel that's misrepresenting, you could also add:

> /n/n(Buf can also be used for other purposes, but currently Pants only uses it as a linter.)